linera-execution/src/policy.rs

```diff
@@ -97,51 +126,38 @@ impl ResourceControlPolicy {
     /// This can be used in tests that need whole numbers in their chain balance and don't expect
     /// to execute any Wasm code.
     pub fn only_fuel() -> Self {
-        ResourceControlPolicy {
-            certificate: Amount::ZERO,
-            fuel: Amount::from_atto(1_000_000_000_000),
-            storage_num_reads: Amount::ZERO,
-            storage_bytes_read: Amount::ZERO,
-            storage_bytes_written: Amount::ZERO,
-            storage_bytes_stored: Amount::ZERO,
-            maximum_bytes_read_per_block: u64::MAX / 2,
-            maximum_bytes_written_per_block: u64::MAX / 2,
-            messages: Amount::ZERO,
+        Self {
+            fuel_unit: Amount::from_atto(1_000_000_000_000),
+            ..Self::default()
         }
     }
 
     #[cfg(any(test, feature = "test"))]
-    /// Creates a pricing with no cost for anything except fuel, and 0.001 per certificate.
+    /// Creates a pricing with no cost for anything except fuel, and 0.001 per block.
```

Comment:
```suggestion
    /// Creates a policy with no cost for anything except fuel, and 0.001 per block.
```

(Also for the other constructors.)